roles/mailserver/files/docker-compose.yml

```diff
@@ -126,8 +126,37 @@ services:
       - OPENCART_HOST=store.{{ domain_name }}
       - OPENCART_USERNAME=opencart
       - OPENCART_PASSWORD=changeThis
-      - SMTP_HOST=mailserver
-      - SMTP_PORT=25
+      - SMTP_HOST=mailserver 
+      - SMTP_PORT=25 
+      - SMTP_USER=username@domain.ext
+      - SMTP_PASSWORD=password
+{% endif %}
+
+{% if use_poweradmin %}
+  poweradmin:
+    image: secns/pdns
+    container_name: poweradmin
+    restart: on-failure
+    depends_on:
+      - mariadb
+    restart: on-failure
+    volumes:
+      - /mnt/docker/poweradmin:/poweradmin
+    ports:
+    - "53:53"
+    - "53:53/udp"
+    environment:
+    - PDNS_ALLOW_AXFR_IPS=127.0.0.1
+    - PDNS_DISTRIBUTOR_THREADS=3
+    - PDNS_CACHE_TTL=20
+    - PDNS_RECURSIVE_CACHE_TTL=10
+    - DB_ENV_MYSQL_ROOT_PASSWORD=changeme
+    - MARIADB_PASSWORD={{ db_password }}
+    - MYSQL_ROOT_PASSWORD=changeme
+  db:
+    image: mysql
+    environment:
+    - MYSQL_ROOT_PASSWORD=changeme
```

Comment:
We really don't want or need a second SQL database running on the host. Should user the existing `mariadb` setup.